java/src/test/java/com/google/oak/client/OakClientTest.java

```diff
@@ -62,15 +74,25 @@ public Result<AttestationBundle, String> getEvidence() {
 
     @Override
     public Result<byte[], String> invoke(byte[] requestBytes) {
-      // TODO(#3642): Use hybrid encryption for requests and responses.
-      EncryptedResponse encryptedResponse =
-          EncryptedResponse.newBuilder()
-              .setEncryptedMessage(AeadEncryptedMessage.newBuilder()
-                                       .setCiphertext(ByteString.copyFrom(TEST_RESPONSE))
-                                       .setAssociatedData(ByteString.copyFrom(TEST_ASSOCIATED_DATA))
-                                       .build())
-              .build();
-      return Result.success(encryptedResponse.toByteArray());
+      Result<Encryptor.DecryptionResult, Exception> decryptRequestResult =
+          serverEncryptor.decrypt(requestBytes);
+      if (decryptRequestResult.isError()) {
```

Comment:
Since the function returns a `Result`, please use the functional API, instead of explicitly calling `isError` and `get`.

In particular you should be able to do something like:
```
return result
   .map(decryptedRequest -> doStuffWith(decryptedRequest))
   .mapErr(err -> convertError(err));
```

Let me know if you run into any issues with it :)

Reply:
Done